UDV-Core/src/Extensions/Contribute/src/ContributeController.js

```diff
@@ -213,6 +238,108 @@ export function ContributeController(documentController){
     }
   }
 
+  /**
+   *   Updates document data
+   */
+  //=============================================================================
+  this.documentUpdate = function documentUpdate(){
+
+    this.updatedData = new FormData(document.getElementById('updateForm'));
```

Comment:
Avoid hardcoded dependencies when possible: Can you retrieve 'updateForm' from the documentUpdate attribute ?

Reply:
Good point. I did the same for the "filterForm" and the "creationForm"